### DISCUSSION TITLE
Manual entry of hours

### DISCUSSION THREAD

**Discussion by BoxOfSnoo** [solidtime-io/solidtime#239] [Feature requests]
When creating a time entry, I would find it very useful to just say "3 hours" for a task, and have it figure out the rest.  Clockify does this by picking the current time as a start time and adding the end time on to that.

It would also help get around a very frustrating "24h format only" time entry window.  And possibly help me avoid having to set a date for start **and** end - (100% of my time entries are on the same day... I'm not sure how common it is for people to bill through several days?  I suppose maybe it's to keep things more generic, for non-billable jobs.)

Thanks, really looking forward to see how this app grows.

**Comment by Onatcer** [comment]
thanks for the feedback. you basically mean the functionality that already exists in the main timetracker component on the dashboard / timer view, right? but for the manual time entries

**Comment by BoxOfSnoo** [comment]
There is an entry field? Where?

**Comment by Onatcer** [comment]
you can type in natural language in this field here 
<img width="824" alt="image" src="https://github.com/user-attachments/assets/ca70e7e6-7698-43a6-b670-ec5ca898d14a">
is that not what you mean? it is not supported in the manual time entry modal yet tho.

**Comment by BoxOfSnoo** [comment]
OK, to be clear, that field is incredibly undiscoverable.  The "What are you working on?" one is more obvious because of the template text but a bit of border around the fields or bigger contrast in the shading would help a lot.

**Comment by Onatcer** [comment]
I'd appreciate it if you toned down your comments a bit, I'm happy to listen to feedback and improve upon it, but there is simply no need to call something "incredibly undiscoverable". 

I thought Clockify would work the same way as Toggl here, apparently it does not, it looks like you can only click on the time duration field after it is started. I'm aware that it is not very clear currently that you can actually type natural language in the duration field and I plan to add a tooltip in the future to explain that you can type natural language in there. I didn't think the field in general was hard to find but I think the border is a good suggestion I'll try, if it doesn't look too cluttered I'll do that because it will also make it easier to add a better focus-visible state there.

**Comment by BoxOfSnoo** [comment]
Sorry, I didn’t think that was harsh, certainly not my intention. I had just finished entering a whole pile of entries using the manual box and it was quite slow without realizing that was right in front of me.

Thanks for your work on this.

**Comment by korridor** [comment]
We decided to move feature requests into github discussions, to separate feature requests from bugs and allow for better prioritization. If this feature is important to you, you can upvote it to let us know.
